### ISSUE TITLE
autocomplete="off" on inputs

### ISSUE THREAD

**Issue by alxpereira** [charliekassel/vuejs-autocomplete#46]
Hi, 

Is there any possibility to pass the option to deactivate the default chrome autocompletion ? 
When i'm passing an additional param as autocomplete="off", the attribute is getting passed on the parent <div> but not on the input. 

Thanks !

**Comment by eddit** [comment]
Yes please. Having the component auto-complete and then the browser autocomplete on top of each other is causing issues.
